README.md

```diff
@@ -81,6 +81,25 @@ Some empty `<% include %>` placeholders are included to let you easily add more
 
 The BetterNavigator.ss template's scope is set to the page that is being viewed, so any methods available in your page controller will be available in the BetterNavigator.ss template. This should allow you to add custom links by page type and introduce complex logic if you want to.
 
+## Overriding the "Edit in CMS" Link
+
+There may be occasions when you wish to override the "Edit in CMS" link. For example to point to the edit form for a displayed DataObject, rather than for the Page itself. To do so, simply add a `BetterNavigatorEditLink()` method to your page's Controller, e.g.:
+
+````php
+// EventsPageController.php
+
+/**
+ * Return an alternative URL for the BetterNavigator Edit in CMS link.
+ * @return string
+ */
+public function BetterNavigatorEditLink()
+{
+    return CMSEditLinkAPI::find_edit_link_for_object($this->displayedEvent());
```

Comment:
Wonder if we should add a canEdit check here for consistency with the regular behaviour? It doesn't provide any extra security of course, but could stop someone following a link to the CMS and then getting a permission error.

```suggestion
    $event = $this->displayedEvent();
    return $event->canEdit() ? CMSEditLinkAPI::find_edit_link_for_object($event) : false;
```